docs/pipelines/index.md

```diff
@@ -134,3 +134,199 @@ else
 <!-- endSnippet -->
 
 Use `ExecuteOutcomeAsync(...)` in high-performance scenarios where you wish to avoid re-throwing exceptions. Keep in mind that Polly's resilience strategies also make use of the `Outcome` struct to prevent unnecessary exception throwing.
+
+## Diagrams
+
+### Variant one
+
+<!-- snippet: resilience-pipeline-diagram-variant-1 -->
```

Comment:
Let's put the text (or different text) before the snippets, rather than going straight into a code snippet.